File: 2-semestre/uri-online-judge/javascript/1180.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const n = parseInt(lines.shift());
const x = lines.shift().split(" ").map(Number);
const m = Math.min(...x);

console.log(`Menor valor: ${m}\nPosicao: ${x.indexOf(m)}`);

File: 2-semestre/uri-online-judge/c/1180.c
#include <stdio.h>
 
int main() {
    int n, m = 0, p = 0, i;

    scanf("%d", &n);

    int x[n];

    for (i = 0; i < n; i++) {
        scanf("%d", &x[i]);
    }

    m = x[0];

    for (i = 0; i < n; i++) {
        if (x[i] < m) {
            m = x[i];
            p = i;
        }
    }

    printf("Menor valor: %d\nPosicao: %d\n", m, p);

    return 0;
}
